recipes/accelerator-toolbox/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "accelerator-toolbox" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/accelerator_toolbox-{{ version }}.tar.gz
+  sha256: 9f4c67216634d20b07f8105ead471b7440c80151dfe81ad27538ea4adefdaa89
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win32 or linux32]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools >=64
+    - setuptools-scm >=7
+    - numpy
+    - wheel
+    - pip
+  run:
+    - python
+    - importlib-resources  # [py<39]
+    - {{ pin_compatible('numpy') }}
```

Comment:
Please don't use this anymore. `numpy` has a `run_exports` to set this correctly.
```suggestion
```